packages/stack/src/views/Stack/Card.tsx

```diff
@@ -574,7 +574,13 @@ export default class Card extends React.Component<Props> {
                 this.offset,
                 cond(
                   this.distance,
-                  divide(this.gesture, this.distance),
+                  divide(
+                    multiply(
+                      this.gestureUntraversed,
```

Comment:
This is vital because of some race condition. I'm not fully confident, but this inlining cannot have any negative impact and fix flickering. Flickering became an issue after fix from the previous commit.